server/halomod_app/__init__.py

```diff
@@ -127,10 +127,16 @@ def get_names():
     # can be used to represent a halo model.
     #
     # expects: None
-    # outputs: KEYMAP as defined in `utils.py`
+    # outputs: {
+    #   xLabels: <data from utils.py>,
+    #   plotOptions: <data from KEYMAP in utils.py>
+    # }
     @app.route('/get_plot_types', methods=["GET"])
     def get_plot_types():
-        res = utils.KEYMAP
+        res = {
+            'xLabels': utils.XLABELS,
+            'plotOptions': utils.KEYMAP
+        }
         return jsonify(res)  # returns full key map of plot types
```

Comment:
I think its safe enough to move these constants to the front end and scrap this endpoint. utils.XLABELS and utils.KEYMAP were part of the old halomod web app implementation which is not going to be maintained or updated anymore. As a result, these will be stable and there shouldn't be a need to pull them from the server dynamically. If steven wants to update them he'll need to do it manually regardless of whether its on the client or the server.

Would you be open to having the download functionality on the client send X & Y labels with the request? Then you can keep it JUST on the client and not have to worry about it during startup anymore!

Reply:
It sounds like these things should be included in the NEW US. Like we had discussed at the meeting: https://the-halo-mod.atlassian.net/jira/software/projects/HM/boards/1?selectedIssue=HM-278

Reply:
Unresolving so @aschnapp can see. I think this is fair game for either US because this is used by both approaches to axis selection.

The basic point is that utils.KEYMAP and utils.XLABEL are more tied to the display side of things and not the halomod side of things. I think it makes sense to have these constants on the client-side, instead, as a result. If we get it there it also simplifies the back and forth quite a bit because you just need to include the X and Y axis options in the request to the server's getPlot and don't have to duplicate data on the client and server. It also removes the need for:
- getPlotTypes in store
- getPlotTypes on server
- the need to store constants (plot types) in our store
- etc...

I think this is a fairly simple refactor with decent payoff in code clarity so if it doesn't make it on this PR I will definitely want it on Alex's. Maybe you two can figure out who should be responsible for it since its so closely tied to both user stories?